merlin/study/batch.py

```diff
@@ -135,6 +135,11 @@ def get_node_count(default=1):
     :param returns: (int) The number of nodes to use.
     """
 
+    # Flux version check
+    flux_ver = get_flux_version(parsed_batch["flux exe"], no_errors=True)
+    if int(flux_ver.split(".")[1]) < 17:
```

Comment:
Need a check to make sure versions 1+.X.X are ok
Like major, minor, _ = flux_ver.split(‘.’)
If major < 1 and minor < 17 then error

Reply:
ahh yes good catch let me go fix that